docs/car-park-skills.md

```diff
@@ -1,16 +1,21 @@
-The full Fetch.ai car park AEA demo is documented in its own repo [here](https://github.com/fetchai/carpark_agent).
-
-This demo allows you to test the AEA functionality of the car park AEA demo without the detection logic.
-
-It demonstrates how the AEAs trade car park information.
+The AEA car-park skills demonstrate an interaction between two AEAs.
 
+* The carpark_detection AEA provides data.
```

Comment:
provides information on the number of car parking spaces available in a given vicinity.